.github/workflows/main.yml

```diff
@@ -26,6 +25,42 @@ jobs:
       - name: Create executable
         run: pkg package.json
 
+      - uses: actions/upload-artifact@main
+        with:
+          name: Windows Artifacts
+          path: dist/ao-loot-logger-node12-win.exe
+
+
+  linux-build-publish:
+    runs-on: ubuntu-18.04
+    needs: windows-build
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+
+      - name: Install libpcap
+        run: sudo apt-get install libpcap-dev -y
+
+      - name: Install pkg
+        run: npm install -g pkg
+
+      - name: Install dependecies
+        run: npm install
+
+      - name: Create executable
+        run: pkg package.json
```

Comment:
Let's use [`---targets node14-linux-x64` ](https://github.com/vercel/pkg#targets) when building the linux executable and [`---targets node12-win-x64` ](https://github.com/vercel/pkg#targets) for windows.

This should cut the build time and remove `Remove windows executable` step.